examples/pytorch/audio-classification/README.md

```diff
@@ -0,0 +1,152 @@
+    --save_strategy epoch \
+    --load_best_model_at_end True \
+    --save_total_limit 3 \
+    --seed 0 \
+    --push_to_hub \
+    --push_to_hub_model_id wav2vec2-base-keyword-spotting
+```
+
+On a single V100 GPU (16GB), this script should run in ~10 minutes and yield accuracy of **98.4%**.
+
+## Multi-GPU 
+
+Here we show how to fine-tune [wav2vec2-base](https://huggingface.co/facebook/wav2vec2-base) for 🌎 **Language Identification** on the [CommonLanguage dataset](https://huggingface.co/datasets/beans).
+
+👀 See the results here: [anton-l/wav2vec2-base-langid](https://huggingface.co/anton-l/wav2vec2-base-langid)
+
+```bash
+python run_audio_classification.py \
+    --model_name_or_path facebook/wav2vec2-base \
+    --dataset_name anton-l/common_language \
+    --audio_column_name path \
+    --label_column_name language \
+    --output_dir ./wav2vec2-base-langid \
+    --overwrite_output_dir \
+    --remove_unused_columns False \
+    --do_train \
+    --do_eval \
+    --fp16 \
+    --learning_rate 3e-5 \
+    --max_length_seconds 16 \
+    --warmup_ratio 0.1 \
+    --num_train_epochs 10 \
+    --per_device_train_batch_size 8 \
+    --gradient_accumulation_steps 4 \
+    --per_device_eval_batch_size 1 \
+    --dataloader_num_workers 8 \
+    --logging_strategy steps \
+    --logging_steps 10 \
+    --evaluation_strategy epoch \
+    --save_strategy epoch \
+    --load_best_model_at_end True \
+    --save_total_limit 3 \
+    --seed 0 \
+    --push_to_hub \
+    --push_to_hub_model_id wav2vec2-base-langid
+```
+
+On 4 V100 GPUs (16GB), this script should run in ~1 hour and yield accuracy of **77.51%**.
+
+## Using your own data
+
+To use your own dataset, convert your data into a `csv` or `json` format with the 
+fields `file` and `label` like so:
+
+```json lines
+{"file": "/absolute/path/to/sample0.wav", "label": "cat"}
+{"file": "/absolute/path/to/sample1.wav", "label": "dog"}
+{"file": "/absolute/path/to/sample2.wav", "label": "bird"}
+```
+
+Once you've prepared your dataset, you can run the script like this:
+
+```bash
+python run_audio_classification.py \
+    --dataset_name my-own-dataset \
+    --train_file <path-to-train-file> \
+    --validation_file <path-to-validation-file> \
+    --output_dir ./outputs/ \
+    --remove_unused_columns False \
+    --do_train \
+    --do_eval
+```
+
+
+## Sharing your model on 🤗 Hub
+
+0. If you haven't already, [sign up](https://huggingface.co/join) for a 🤗 account
+
+1. Make sure you have `git-lfs` installed and git set up.
+
+```bash
+$ apt install git-lfs
+$ git config --global user.email "you@example.com"
+$ git config --global user.name "Your Name"
+```
+
+2. Log in with your HuggingFace account credentials using `huggingface-cli`
+
+```bash
+$ huggingface-cli login
+# ...follow the prompts
+```
+
+3. When running the script, pass the following arguments:
+
+```bash
+python run_audio_classification.py \
+    --push_to_hub \
+    --push_to_hub_model_id <name-your-model> \
```

Comment:
The right argument is `hub_model_id` now, and it should be the full model name (e.g. username/model_id)